### DISCUSSION TITLE
Print counter

### DISCUSSION THREAD

**Discussion by ghost** [Jyers/Marlin#776] [General Discussion]
The print counter is not apearing on the rpecomplide manual meshh 5x5 4.2.2

**Comment by Jyers** [comment]
Just to check, you're looking in the info menu correct? Also have you tried resetting and then storing eeprom?

**Reply by ghost** [reply]
Yes, i tired resetting de eeprom. I flashed the manual mesh 5x5 4.2.2. The info menu is under the conreol menu. When you open it just days the link of the github, the pirnt volume and the version of the firmware. 
Maybe its the fact that the print vounter is activated just on the info menu of the main menu? And since i have the manual mesh one it doesnt aoear.

**Reply by Jyers** [reply]
It should be there no matter what, I'll check the configs to make sure I didn't miss one

**Reply by ghost** [reply]
When i finish this print i will fouble check and if its not there i will attach a picture

**Reply by ghost** [reply]
Well i etither flashed the wrong build or didi before 1.3.2 but its definitly 1.3.1. Im so Sorry.

**Comment by Tuncay-Ayhan** [comment]
> The print counter is not apearing on the rpecomplide manual meshh 5x5 4.2.2

Remember that when you flash the firmware, you have to rename the file. Maybe you got excited and thought you flashed it, but it just booted instead of flashing and then booting.

**Reply by ghost** [reply]
Well, i think its ok,. Because u can turn on the printer without the sd card and it works. I did it a week ago. First i tried flashing it directly to the manual mesh but fidnt work, the screen will stay black for 30mins and not change. The i used the precompiled defualt version and it worked. So i the agian flashed the manual mesh and was fine. Also, sorry for my English, not my first language. But problem is not that it doesnt count ir, its just not there the counter. It only displays the build volume, the firmware version and the github page.

**Reply by Tuncay-Ayhan** [reply]
> Well, i think its ok,. Because u can turn on the printer without the sd card and it works. I did it a week ago. First i tried flashing it directly to the manual mesh but fidnt work, the screen will stay black for 30mins and not change. The i used the precompiled defualt version and it worked. So i the agian flashed the manual mesh and was fine. Also, sorry for my English, not my first language. But problem is not that it doesnt count ir, its just not there the counter. It only displays the build volume, the firmware version and the github page.

_**Because u can turn on the printer without the sd card and it works.**_ 
It is suppose to work without a sd card in it. 

_**First i tried flashing it directly to the manual mesh but fidnt work,**_
What do you mean by that?

_**the screen will stay black for 30mins and not change**_
It stayed black for 30 minutes when you inserted the SD card with the .BIN file (with an unique name)?

_**The i used the precompiled defualt version and it worked. So i the agian flashed the manual mesh and was fine.**_
Can you confirm you flashed the firmware correctly? What version does it say when you check the **info** section?

_**But problem is not that it doesnt count ir, its just not there the counter. It only displays the build volume, the firmware version and the github page.**_
This tells me that you didn't successfully flash the newer firmware. Under info, it should say that you are on 1.3.2 at least, because before that, there was no Counter information available on the LCD.

Just to be sure. Re-format you SD card as FAT32 and use this picture as a guide. 
![fat_32_cluster_sizes](https://user-images.githubusercontent.com/28215053/117853451-b8c2c280-b288-11eb-93e3-c2d43b1cc89b.png)

Then download a new copy of the latest firmware. Rename it to anything you want. Call it for example **20210511.bin** . Copy it to your SD card, turn off you printer. Insert card. Turn printer on, screen should stay black for 5/6 seconds and then start to boot. 

Then go to **info** , check the version number. Should say 1.3.2

You can remove your card now. As you won't need it anymore. For example, if you now redo the whole thing as I mentioned and leave the firmware name as is. You'll see it doesn't do anything different then booting a couple of seconds faster.

**Reply by ghost** [reply]
I understand, to avoid any confusion i will re answer your questions once i am done printing, in abaount 6hs. 
What i was saying tho, is that when i first try to flash the 5x5 manual mesh the printer wont boot. So i tried the default one and worked, and finally tryed the manual mesh again.
But as i said, will check the version when it finishes

**Reply by ghost** [reply]
Well i etither flashed the wrong build or did it before 1.3.2 but its definitly 1.3.1. Im so Sorry.

**Reply by Tuncay-Ayhan** [reply]
> Well i etither flashed the wrong build or did it before 1.3.2 but its definitly 1.3.1. Im so Sorry.
> 
> 

No worries, now you can flash the right one and be good. 😀
